docs/gateway-api/gateway-api-v1.md

```diff
@@ -165,7 +165,153 @@ In AKO Gateway API Implementation, Gateway objects corresponds to following AVI
+        requestHeaderModifier:
+          add:
+          - name: request-header
+            value: test-request-header
+          remove:
+          - test-remove-header
+  ```
+
+For the above yaml, an HTTPPolicySet with a `HTTP Request Rule ` with action as `Modify Header` having `Add Header -> Header Name` as `request-header` and `Header Value` as `test-request-header` and `Remove Header -> Header Name` as `test-remove-header` will be added to the childVS corresponding to the rule. When the request comes for host and path `products.avi.internal/foo` , a header with name as `request-header` and value as `test-request-header` will be added to the request before it is being sent to the backend server. Also , if there is any header with name "test-remove-header" is present, it will be removed.
+
+#### RequestRedirect:
+
+HTTPRoute RequestRedirect Filter in AKO Gateway API implementation is supported using HTTPPolicy Sets `Redirect` action. Currently we support only `hostname` and `statusCode` redirection. Once, a `RequestRedirect` fllter is found in a rule in an HTTPRoute, AKO will add an httppolicyset with HTTPRequestRule and action as `Redirect` to the childVS corresponding to the rule. Whenever the traffic lands on that child VS, a response with the `redirect hostname` and `statusCode` as specified in the HTTPRoute RequestRedirect filter spec will be sent back.
+
+ A sample httproute with RequestRedirect filter is shown below:
+
+  ```yaml
+    apiVersion: gateway.networking.k8s.io/v1
+    kind: HTTPRoute
+    metadata:
+      name: insecure-http-route
+      namespace: test-httproute-ns
+    spec:
+      parentRefs:
+      - name: test-insecure-gateway
+        sectionName: http
+      hostnames:
+      - "products.avi.internal"
+      rules:
+      - matches:
+        - path:
+            value: "/foo"
+        filters:
+        - type: RequestRedirect
+          requestRedirect:
+            statusCode: 302
+            hostname: "apps.avi.internal"
+  ```
+
+For the above yaml, an HTTPPolicySet with a rule with action as `Redirect` with `Redirect-> Status Code` as `302` and `Redirect-> Host` as `apps.avi.internal` will be added to the childVS corresponding to the rule. When the request comes for host and path `products.avi.internal/foo` , redirect host will be set to `apps.avi.internal` and response code will be set to `302` before the response is sent back to the client.
+
+#### UrlRewrite:
+
+HTTPRoute urlRewrite Filter in AKO Gateway API implementation is supported using HTTPPolicy Sets `Rewrite URL` action. Currently we support only `hostname` and `path` rewrite. Once, a `urlRewrite` fllter is found in a rule in an HTTPRoute, AKO will add an httppolicyset with HTTPRequestRule and action as `Rewrite URL` to the childVS corresponding to the rule. Whenever the traffic lands on that child VS, the path and the hostname will changed to the rewtite path and hostname specified in the HTTPRoute URLRewrite filter specification.
```

Comment:
Can we add it as `Note` what config options from urlRewrite of HTTPRoute is supported by AKO

Reply:
Already added as HTTPRoute Limitation on line number 473 and 474